tests/py/test_fake_data.py

```diff
@@ -34,3 +33,12 @@ def test_fake_participant_identity(self):
         crusher = self.make_participant('crusher', email_address='crusher@example.com')
         country_id = fake_data.fake_participant_identity(crusher)
         assert [x.country.id for x in crusher.list_identity_metadata()] == [country_id]
+
+    def test_fake_team(self):
+        """
+            Just a test to ensure that fake_team does not fail for 
+            teams with apostrophes
+        """
+        crusher = self.make_participant('crusher', email_address='crusher@example.com')
+        team = fake_data.fake_team( self.db, crusher, "D'Amorebury") 
+        assert team.name != "d-amorebury"
```

Comment:
Well, if the test is called `test_fake_team_does_not_fail_for_teams_with_apostrophes` then the assertion is sufficient. :)